webpack/scenes/ContentViews/Details/Filters/ContentViewFilterDetailsHeader.js

```diff
@@ -11,10 +11,16 @@ const ContentViewFilterDetailsHeader = ({
   cvId, filterId, details, setShowAffectedRepos,
 }) => {
   const {
-    type, name, inclusion, description,
+    type, name, inclusion, description, rules,
   } = details;
+  let errataByDate = false;
+  if (type === 'erratum' && rules[0]?.types) errataByDate = true;
```

Comment:
Since you'll likely be importing lodash for below, using head here would be good.